pages/volunteers.tsx

```diff
@@ -85,7 +85,7 @@ const VolunteerPage = () => {
             alignItems: 'center',
             gap: { xs: 4, md: '40px' },
           }}
-          maxWidth={'880px'}
+          maxWidth={'1000px'}
```

Comment:
per Kylie's suggestion it should be 880 https://www.figma.com/file/Qa1UC7cGLmEXbWXuRmgjpO/OS-Website-Annotations?type=whiteboard&node-id=136-244&t=R7Ic16ISjJQafpE5-0

setting it to 1000px would make the hero section not align with the content of the page.

Reply:
I would suggest getting the max width right from the theme file! As in `theme.breakpoints.values.lg`. Doing that on all pages' maxWidths would help w/ consistency